cmd/nerdctl/restart_linux_test.go

```diff
@@ -42,3 +44,28 @@ func TestRestart(t *testing.T) {
 	newPid := newInspect.State.Pid
 	assert.Assert(t, pid != newPid)
 }
+
+func TestRestartPIDContainer(t *testing.T) {
+	t.Parallel()
+	base := testutil.NewBase(t)
+
+	baseContainerResult := base.Cmd("run", "-d", testutil.AlpineImage, "sleep", "infinity").Run()
+	baseContainerID := strings.TrimSpace(baseContainerResult.Stdout())
```

Comment:
Using `testutil.Identifier(t)` as container name.

Reply:
I know that the run command with `-d` should print a container's id.
Why should we name for it?

Reply:
Ask in turn, why do we need to parse the id from stdout?